pkg/chartutil/create.go

```diff
@@ -47,10 +47,14 @@ const (
 	DeploymentName = TemplatesDir + sep + "deployment.yaml"
 	// ServiceName is the name of the example service file.
 	ServiceName = TemplatesDir + sep + "service.yaml"
+	// ServiceAccountName is the name of the example serviceaccount file.
+	ServiceAccountName = TemplatesDir + sep + "serviceaccount.yaml"
 	// NotesName is the name of the example NOTES.txt file.
 	NotesName = TemplatesDir + sep + "NOTES.txt"
 	// HelpersName is the name of the example NOTES.txt file.
```

Comment:
:s/HelpersName is the name of the example NOTES.txt file./HelpersName is the name of the example NOTES.txt file./HelpersName is the name of the example helpers file.